fetchai/ledger/api/tx.py

```diff
@@ -76,14 +76,14 @@ def __init__(self,
         self.valid_until = valid_until
         self.charge = charge
         self.charge_limit = charge_limit
-        self.transfers = {Address(t['to']): t['amount'] for t in transfers}
+        self.transfers = [{Address(t['to']): int(t['amount'])} for t in transfers]
         self.signatories = signatories
         self.data = data
 
     def transfers_to(self, address: AddressLike) -> int:
         """Returns the amount of FET transferred to an address by this transaction, if any"""
         address = Address(address)
-        return self.transfers.get(address, 0)
+        return [t[address] for t in self.transfers if address in t]
```

Comment:
First, returning list instead of accumulated scalar value is change of interface, thus it might be a bit controversial, and might need to be revised (see the note bellow).
Second, the spec for return type `-> int` is no more correct.
NOTE: 
If this his api (`Tx.transfers_to(...)`) is already used by external projects, then this is no-go, and I will change it to returning accumulated single scalar value.
I haven't found any code (in repositories owned by our Fetch.AY GitHub organisation) which would be using it except [this commented-out code](https://github.com/fetchai/agents-aea/blob/bf4badf956ce1418c25f5377b07fa3e2a13f8f9e/aea/crypto/ledger_apis.py#L214)